src/engine/modules.js

```diff
@@ -179,7 +179,9 @@ function loadImports(self, imports) {
   }
   for (let i = 0; i < imports.length; ++i) {
     const [, moduleName, moduleVersion, moduleAlias] = imports[i];
-    if (typeof moduleVersion === "undefined") continue;
+    if (typeof moduleVersion === "undefined" || moduleVersion === null) {
```

Comment:
Yes, it looks like `import Something 0` is permitted.

Perhaps `typeof moduleVersion !== 'number'`? That should be easier to read, and the code below relies on `moduleVersion` to be a `number`.